google/cloud/documentai_toolbox/wrappers/document.py

```diff
@@ -722,27 +699,35 @@ def split_pdf(self, pdf_path: str, output_path: str) -> List[str]:
         return output_files
 
     def convert_document_to_annotate_file_response(self) -> AnnotateFileResponse:
-        r"""Convert OCR data from Document.proto to AnnotateFileResponse.proto for Vision API.
+        r"""Convert OCR data from `Document.proto` to `AnnotateFileResponse.proto` for Vision API.
 
         Args:
             None.
         Returns:
             AnnotateFileResponse:
-                Proto with TextAnnotations.
+                Proto with `TextAnnotations`.
         """
-        return _convert_to_vision_annotate_file_response(self.text, self.pages)
+        return AnnotateFileResponse(
+            responses=[
+                vision_helpers.convert_page_to_annotate_image_response(
+                    docai_page, self.text
+                )
+                for shard in self.shards
+                for docai_page in shard.pages
+            ]
+        )
 
     def convert_document_to_annotate_file_json_response(self) -> str:
-        r"""Convert OCR data from Document.proto to JSON str of AnnotateFileResponse for Vision API.
+        r"""Convert OCR data from `Document.proto` to JSON str of `AnnotateFileResponse` for Vision API.
 
         Args:
             None.
         Returns:
             str:
-                JSON string of TextAnnotations.
+                JSON string of `TextAnnotations`.
         """
         return AnnotateFileResponse.to_json(
-            _convert_to_vision_annotate_file_response(self.text, self.pages)
+            self.convert_document_to_annotate_file_response()
         )
 
     def export_images(
```

Comment:
The method name here seems to suggest that it exports all images, but actually only those that belong to entities?

(Perhaps all images belong to entities?)

Reply:
Note - DocAI doesn't do any general image recognition currently, it's only for specific entity extraction